### PR TITLE
New package: cloudflare-warp-2023.3.398

### PR DESCRIPTION
**Pull request by 2trvl**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/cloudflare-warp/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'cloudflare-warp'
+pkgname=cloudflare-warp
+version=2023.3.398
+revision=1
+archs="i686 x86_64"
+short_desc="Cloudflare Warp Client"
+maintainer="Andrew Shteren <dank.txt@gmail.com>"
+license="Unknown"
```
Comment (Duncaen) [inline srcpkgs/cloudflare-warp/template:8]: This should probably be something like `custom: Proprietary`.

File: srcpkgs/cloudflare-warp/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'cloudflare-warp'
+pkgname=cloudflare-warp
+version=2023.3.398
+revision=1
+archs="i686 x86_64"
```
Comment (Duncaen) [inline srcpkgs/cloudflare-warp/template:5]: You are only downloading and packaging `x86_64` binaries.

File: srcpkgs/cloudflare-warp/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+short_desc="Cloudflare Warp Client"
+maintainer="Andrew Shteren <dank.txt@gmail.com>"
+license="Unknown"
+homepage="https://1.1.1.1/"
+repository="nonfree"
+restricted="yes"
+nostrip="yes"
+
+_debpkgid="cloudflare_warp_2023_3_398_1_amd64_002e48d521.deb"
```
Comment (Duncaen) [inline srcpkgs/cloudflare-warp/template:14]: `${version//./_}` instead of hard coding the version, maybe also add the `1` to the version.

**Review by Duncaen — CHANGES_REQUESTED** [review]
The whole logging mechanism diverts from any other service packaged and I don't think this is necessary. Just use `vlogger(1)` in the log service and let the user decide to log messages in any other way.

Commits should be squashed and follow the format in https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md.

**Comment by 2trvl** [comment]
bump

**Pull request closed without merging**